dotCMS/src/main/java/com/dotcms/jobs/business/processor/impl/ImportContentletsProcessor.java

```diff
@@ -397,10 +402,12 @@ private Map<String, List<String>> processImport(final boolean preview, final Job
                 preview ? "Preview" : "Process"));
         Logger.info(this, String.format("-> Content Type: %s", contentType.variable()));
 
-        return ImportUtil.importFile(importId, currentSiteId, contentType.id(), fields, preview,
+        return ImportUtil.importFileResult(importId, currentSiteId, contentType.id(), fields,
```

Comment:
Saml thing here, would use a bean with a builder pattern

Reply:
This is related with how the `ImportUtil` already works and it is outside the scope of the ticket.